grafana-plugin/src/pages/integration/Integration.tsx

```diff
@@ -834,164 +834,167 @@ const IntegrationActions: React.FC<IntegrationActionsProps> = ({
           </Button>
         </WithPermissionControlTooltip>
 
-        <WithContextMenu
-          data-testid="integration-settings-context-menu"
-          renderMenuItems={() => (
-            <div className={cx('integration__actionsList')} id="integration-menu-options">
-              <div className={cx('integration__actionItem')} onClick={() => openIntegrationSettings()}>
-                <Text type="primary">Integration Settings</Text>
-              </div>
-
-              {store.hasFeature(AppFeature.Labels) && (
-                <WithPermissionControlTooltip userAction={UserActions.IntegrationsWrite}>
-                  <div className={cx('integration__actionItem')} onClick={() => openLabelsForm()}>
-                    <Text type="primary">Alert group labels</Text>
-                  </div>
-                </WithPermissionControlTooltip>
-              )}
+        <div data-testid="integration-settings-context-menu-wrapper">
```

Comment:
This diff looks big but in reality it is just wrapper <WithContextMenu /> with a div